Makefile.build

```diff
@@ -630,15 +627,65 @@ test-suite: world byte $(ALLSTDLIB).v
 # Default rules for compiling ML code
 ###########################################################################
 
-# Target for libraries .cma and .cmxa
+gramlib__pack:
+	mkdir -p $@
+
+# gramlib.ml contents
+gramlib__pack/gramlib.ml: gramlib__pack
+	echo " \
+module Ploc    = Gramlib__Ploc \
+module Plexing = Gramlib__Plexing \
+module Gramext = Gramlib__Gramext \
+module Grammar = Gramlib__Grammar" > $@
+
+gramlib__pack/gramlib__Ploc.mli: gramlib/ploc.mli gramlib__pack
+	cp -a $< $@
```

Comment:
Please put in the file rename directive (the thing `copy#` does in dune).

Reply:
OK